libs/workspace-manager/src/lib/workspace-manager/workspace-manager.routes.ts

```diff
@@ -0,0 +1,32 @@
+import { Routes } from '@angular/router';
+import { WorkspaceManagerComponent } from '@angular-cli-gui/workspace-manager';
+
+export const WORKSPACE_MANAGER_ROUTES: Routes = [
```

Comment:
choose a naming convention and stick with it (before you had `appRoutes`)

Reply:
updated